environment.yml

```diff
@@ -5,5 +5,7 @@ channels:
  - defaults
 dependencies:
  - mamba
- - screed>=1.0.5,<2
- - sourmash>=4.1.1,<5
```

Comment:
crazy that sourmash isn't needed in the base env...did a scan through the snakefile tho and none of the rules need sourmash outside of conda env `env/sourmash.yml`!

Reply:
yep! I could actually remove lxml, too, by providing it in a conda environment; but I decided not to bother for this PR.